config.toml

```diff
@@ -47,6 +44,22 @@ host = '127.0.0.1'
 # by the telemetry service. Default: 3001
 port = 3001
 
+# The rest section defines parameters for Hermes' built-in RESTful API capabilities.
+# https://hermes.informal.systems/rest.html
+[rest]
+
+# Whether or not to enable the rest service. Default: false
+enabled = true
+
+# Specify the IPv4/6 host over which the built-in HTTP server will serve the restful
+# API requests. Default: 127.0.0.1
+host = '127.0.0.1'
+
+# Specify the port over which the built-in HTTP server will serve the restful API
+# requests. Default: 3000
+port = 3000
+
+
```

Comment:
Two questions:

1. would it be more consistent to move the whole `rest` section above the `telemetry` section? Partly because of the port (3000) which is before the telemetry port (3001), partly because that's the order chosen in the `pub struct Config`.

2. How will we document the REST API in https://hermes.informal.systems/rest.html? Is there an automated way of doing that? (Note we do not want to create the page https://hermes.informal.systems/rest.html yet, we'll create that once we release next ver. of Hermes. Just asking to understand the complexity of documenting the REST API.).

Reply:
1. 👍 
2. I don't think `rouille` provides a way of doing that, so I would suggest doing it by hand for now and if the API either changes to often or its surface increases too much we can revisit this.

Reply:
1. https://github.com/informalsystems/ibc-rs/pull/955/commits/5bafaf302febb6f172242be2b5e6b8beb3c4ec4e